src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -16,7 +16,9 @@ Microsoft.CodeAnalysis.IParameterSymbol.NullableAnnotation.get -> Microsoft.Code
 Microsoft.CodeAnalysis.IPropertySymbol.NullableAnnotation.get -> Microsoft.CodeAnalysis.NullableAnnotation
 Microsoft.CodeAnalysis.ITypeParameterSymbol.ConstraintNullableAnnotations.get -> System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.NullableAnnotation>
 Microsoft.CodeAnalysis.ITypeParameterSymbol.ReferenceTypeConstraintNullableAnnotation.get -> Microsoft.CodeAnalysis.NullableAnnotation
+Microsoft.CodeAnalysis.ITypeSymbol.ToDisplayParts(Microsoft.CodeAnalysis.NullableAnnotation topLevelNullability, Microsoft.CodeAnalysis.SymbolDisplayFormat format = null) -> System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.SymbolDisplayPart>
```

Comment:
>ToDisplayParts [](start = 35, length = 14)

Tagging @gafter as FYI for public API change.

Reply:
API change is no longer present now that https://github.com/dotnet/roslyn/pull/39498 is merged.